engine/src/main/java/org/destinationsol/game/SolGame.java

```diff
@@ -62,10 +64,17 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import java.io.BufferedReader;
+import java.io.File;
 import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
 import java.io.PrintWriter;
 import java.io.UnsupportedEncodingException;
+import java.lang.reflect.Type;
 import java.util.ArrayList;
+import java.util.HashMap;
+
```

Comment:
All of these imports are unused??

Reply:
All imports used.

Reply:
Looks like this might have been a branching quirk - they had been removed on purpose somewhere but may have still been present in this branch pre-merge. I had to resolve conflicts in this file anyway so I removed them again :-)